src/components/Layout.tsx

```diff
@@ -6,7 +6,20 @@ interface LayoutProps {
 
 const Layout = ({ children }: LayoutProps) => {
   return (
-    <Box minHeight={'calc(100vh - (2*64px))'} pb={1}>
+    <Box
+      display='flex'
+      flexDirection='column'
+      flexWrap='wrap'
+      sx={(theme) => ({
+        justifyContent: 'center',
```

Comment:
The idea here was to have search bar in the center when user first visits Musicolor and starts to search an album. But when the results are present, then searchbox and the results keep their previous layout. And that is fine on breakpoints **that are not mobile**. On mobile it looks like this:
<img width="612" alt="Screenshot 2024-02-11 at 14 44 26" src="https://github.com/e1401/musicolor/assets/38010322/d887f745-f9fa-45e6-a538-dc923f97ec99">
so this view needs some rework.

Also, the buttons layout does not match figma and screenshots in the ticket.

One other thing, the branch name for this would be `fix/MC-25--starting-page-redo`, as per previous agreement and naming convention in this humble project